### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by StefanGhiban** [formkit/formkit#247]
### Describe the feature

Hi! This is a great package, but I am wondering if it covers at least the basic accessibility needs of a form. The term itself is missing from the documentation pages and I was curious to see if you have taken this issue into account.

**Comment by dominikklein** [comment]
There is also already this issue: https://github.com/formkit/formkit/issues/123

Maybe you can share your thoughts.
